tests/github/resources/github_conf/empty_collabs/repository_collaborators.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
why change this? :)

Reply:
We have a test for it, but I guess the resource test file was left out

Reply:
it's github, so I will just ask to ensure this is intended here?